scripts/scil_recognize_multi_bundles.py

```diff
@@ -0,0 +1,121 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+"""
+    Compute  RecobundlesX (multi-atlas & multi-parameters).
+    The model need to be cleaned and lightweight.
+    Transform should come from ANTs: (using the --inverse flag)
+    AntsRegistration -m MODEL_REF -f SUBJ_REF
+    ConvertTransformFile 3 0GenericAffine.mat 0GenericAffine.npy --ras --hm
+"""
+
+import argparse
+import logging
+import json
+import os
+import random
+import shutil
+
+import coloredlogs
+import numpy as np
+
+from scilpy.io.utils import add_overwrite_arg, assert_inputs_exist
+from scilpy.segment.voting_scheme import VotingScheme
+
+
+def _buildArgsParser():
+    p = argparse.ArgumentParser(formatter_class=argparse.RawTextHelpFormatter,
+                                description=__doc__)
+
+    p.add_argument('in_tractogram',
+                   help='Input tractogram filename (trk or tck).')
+    p.add_argument('config_file',
+                   help='Path of the config file (json)')
+    p.add_argument('models_directories', nargs='+',
+                   help='Path for the directories containing model.')
+    p.add_argument('transformation',
+                   help='Path for the transformation to model space.')
+
+    p.add_argument('--output_dir', default='voting_results/',
+                   help='Path for the output directory.')
+    p.add_argument('--log_level', default='INFO',
+                   choices=['DEBUG', 'INFO', 'WARNING', 'ERROR'],
+                   help='Log level of the logging class')
+
+    p.add_argument('--multi_parameters', type=int, default=1,
+                   help='Pick parameters from the potential combinations\n' +
+                   'Will multiply the number of time Recobundles is ran.')
+    p.add_argument('--minimal_vote_ratio', type=float, default=0.5,
+                   help='Streamlines will only be considered for saving if\n ' +
+                   'recognized often enough.')
+
+    p.add_argument('--tractogram_clustering_thr',
+                   type=int, default=[12], nargs='+',
+                   help='Input tractogram clustering thresholds ' +
+                   '[%(default)smm].')
+
+    p.add_argument('--processes', type=int, default=1,
+                   help='Number of thread used for computation [%(default)s].')
+    p.add_argument('--seeds', type=int, default=[None], nargs='+',
```

Comment:
Why [None] and not None

Reply:
It is an nargs, so it saves a if later on. Instead of checking if it is None and then I would have to replace it with [None]